packages/server/modules/workspaces/rest/sso.ts

```diff
@@ -119,6 +120,18 @@ router.get(
   }
 )
 
+router.post('/api/v1/auth/logout', async ({ context, res }) => {
```

Comment:
This file already existed before with the same pattern, but you should generally avoid this kind of heavy sideffects in the module root scope, cause all of this will be invoked just by virtue of importing this file, which could theoretically break stuff or cause logic to be invoked way earlier than you anticipate it to.

Generally better to avoid all of this and export a function instead that you then invoke explicitly on Speckle module init

Reply:
Roger, do you mind if I submit this change to you in another PR? There's an unfortunate clash with this and the big SSO PR atm.